tests.webpack.js

```diff
@@ -1,2 +1,2 @@
-const context = require.context('./src', true, /\.test\.js$/)
+const context = require.context('./test', true, /\.test\.js$/)
```

Comment:
I moved `src/index.test.js` to `test/integration.test.js` and added a helper file `test/utils.js`.

What is the purpose of `build:tests:local `?